mysqld-mixin/go.mod

```diff
@@ -0,0 +1,8 @@
+module github.com/prometheus/mysqld_exporter/mysqld_mixin
+
+go 1.14
+
+require (
+	github.com/google/go-jsonnet v0.16.0 // indirect
+	github.com/monitoring-mixins/mixtool v0.0.0-20201004122653-ee7999927a3a // indirect
+)
```

Comment:
Is this expected? Seems like an artifact of `go install`

Reply:
This is there to pin the version of the mixtool and jsonnetfmt we use in CI, so it doesn't break when they change :-)

(This pattern is what we do in node mixin too)